sys/src/libthread/BUILD

```diff
@@ -8,7 +8,7 @@ cc_library(
             "//amd64/include",
         ],
 	srcs = [
-		"amd64.c",
+		"$ARCH.c",
```

Comment:
That regexp is not safe but if you're saying that a $ should be followed by a single {, a set of valid non-} terminated by a }, then you're ok I guess.
${.*} matches ${{{{{{{{{{{{{{{{{{{{{{{{{{{} for example. :-)